server/services/spotify-service.js

```diff
@@ -1,7 +1,7 @@
 const TokenService = require('./token-service')
 const SpotifyWebApi = require('spotify-web-api-node')
 const {clientId, clientSecret, redirectUri} = require('../constants/credentials')
-const scopes = ['playlist-read-private']
+const scopes = ['streaming', 'user-read-birthdate', 'user-read-email', 'user-read-private', 'playlist-read-private']
```

Comment:
Got these from https://beta.developer.spotify.com/documentation/web-playback-sdk/quick-start/

I'm not sure birthdate makes sense but I didn't try with restricted scopes.